chingu-voyage4/Geckos-Team-32
# Created dashboard, setup structure for pages, fixed some styling

## Purpose
* Start coding out front-end with @Raymond-Cox 

## Approach/Fixes
* Setup file structure for pages
* Created static dashboard on left-hand side
* Adjusted nav/footer styles for consistency in code
  * Footer now only appears in tablet/mobile view, in regular desktop view the sidebar replaces the footer


## Testing
* Download this PR to your local machine
* Check to see that a side dashboard appears in desktop view
* Shrink screen to tablet and mobile view and check that the dashboard disappears and that the footer appears

## Notes
* Social media links have dummy paths
* Styles are all temporary
